client/signup/config/flows.js

```diff
@@ -258,15 +258,6 @@ if ( config.isEnabled( 'signup/domain-first-flow' ) ) {
 	};
 }
 
-if ( config.isEnabled( 'signup/social' ) ) {
-	flows.social = {
-		steps: [ 'user-social' ],
-		destination: '/',
-		description: 'Create an account without a blog with social signup enabled.',
-		lastModified: '2017-03-16'
-	};
-}
-
```

Comment:
This flow is still referenced [here](https://github.com/Automattic/wp-calypso/blob/77fde93d3147799d98995592207d21be9ae0e110/client/signup/steps/user/index.jsx#L61).